python/ray/tests/test_basic_2.py

```diff
@@ -649,6 +649,30 @@ def f(arg1, arg2, arg1_duplicate, kwarg1=None, kwarg2=None, kwarg1_duplicate=Non
     arg2 = ray.put([2])
     ray.get(f.remote(arg1, arg2, arg1, kwarg1=arg1, kwarg2=arg2, kwarg1_duplicate=arg1))
 
+    @ray.remote
+    class Actor:
+        def f(
+            self,
+            arg1,
+            arg2,
+            arg1_duplicate,
+            kwarg1=None,
+            kwarg2=None,
+            kwarg1_duplicate=None,
+        ):
+            assert arg1 == kwarg1
+            assert arg1 != arg2
+            assert arg1 == arg1_duplicate
+            assert kwarg1 != kwarg2
+            assert kwarg1 == kwarg1_duplicate
+
+    actor = Actor.remote()
```

Comment:
Could you add a comment saying what you are testing?

Reply:
done

Reply:
nit: `# Test duplicate by-reference arguments on an actor task`

Reply:
I followed the comments above my change.

```python
# Test by-value arguments.
```

```python
# Test by-reference arguments.
```

Since the test case is named `test_duplicate_args`, I think it's OK not to mention "duplicate" again.

Reply:
oh, got it. I missed the test case name.